docs/sphinx/applications/python/krylov.ipynb

```diff
@@ -7,39 +7,45 @@
     "\n",
     "The algorithm works by preparing an initial state, and then defining this state in a smaller subspace constructed with a basis that corresponds to Trotter steps of the initial state. This subspace can be diagonalized to produce an approximate energy for the system without variational optimization of any parameters.\n",
     "\n",
     "In the example below, the initial guess is the ground state of the diagonalized Hamiltonian for demonstration purposes.  In practice one could use a number of heuristics to prepare the ground state such as Hartree Fock or CISD. A very promising ground state preparation method which can leverage quantum computers is the linear combination of unitaries (LCU). LCU would allow for the state preparation to occur completely on the quantum computer and avoid storing an inputting the exponentially large state vector.\n",
     "\n",
     "\n",
-    "Regardless of the method used for state preparation, the procedure begins by selecting a $d$ dimensional basis of reference states ${\\Phi_0 \\cdots \\Phi_d}$ where each is a linear combination of slater determinants. \n",
+    "Regardless of the method used for state preparation, the procedure begins by selecting a $d$-dimensional basis of reference states ${\\Phi_0 \\cdots \\Phi_d},$ where each is a linear combination of Slater determinants: \n",
     "\n",
-    "$$ \\ket{\\Phi_I}  =  \\sum_{\\mu} d_{\\mu I}\\ket{\\phi_{\\mu}} $$\n",
+    "$$ \\ket{\\Phi_I}  =  \\sum_{\\mu} d_{\\mu I}\\ket{\\phi_{\\mu}}. $$\n",
     "\n",
     "\n",
-    "From this, a non-orthogonal Krylov Space $\\mathcal{K} = \\{\\psi_{0} \\cdots \\psi_{N}\\}$ is constructed by applying a family of $s$ unitary operators on each of the $d$ reference states resulting in $d*s = N$ elements in the Krylov space where, \n",
-    "$$ \\ket{\\psi_{\\alpha}} \\equiv \\ket{\\psi_I^{(n)}} = \\hat{U}_n\\ket{\\Phi_I} $$\n",
+    "From this, a non-orthogonal Krylov Space $\\mathcal{K} = \\{\\psi_{0} \\cdots \\psi_{N}\\}$ is constructed by applying a family of $s$ unitary operators on each of the $d$ reference states resulting in $d*s = N$ elements in the Krylov space where \n",
+    "$$ \\ket{\\psi_{\\alpha}} \\equiv \\ket{\\psi_I^{(n)}} = \\hat{U}_n\\ket{\\Phi_I}, $$\n",
     "\n",
     "Therefore, the general quantum state that we originally set out to describe is\n",
     "\n",
-    "$$ \\ket{\\Psi} = \\sum_{\\alpha} c_{\\alpha}\\ket{\\psi_{\\alpha}} = \\sum_{I=0}^d \\sum_{n=0}^s c_I^{(n)}\\hat{U}_n\\ket{\\Phi_I}  $$\n",
+    "$$ \\ket{\\Psi} = \\sum_{\\alpha} c_{\\alpha}\\ket{\\psi_{\\alpha}} = \\sum_{I=0}^d \\sum_{n=0}^s c_I^{(n)}\\hat{U}_n\\ket{\\Phi_I}.  $$\n",
     "\n",
     "The energy of this state can be obtained by solving the generalized eigenvalue problem\n",
-    "$$ \\boldsymbol{Hc}=\\boldsymbol{Sc}E $$\n",
+    "$$ \\boldsymbol{Hc}=\\boldsymbol{Sc}E, $$\n",
     "\n",
-    "Where the elements of the overlap and Hamiltonian matrix are\n",
+    "where the elements of the overlap  are\n",
```

Comment:
Extra space?

Reply:
ah yes, it appears there is an extra space there.